modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Options/Options.es.js

```diff
@@ -515,4 +515,33 @@ describe('Options', () => {
 
 		expect(valueInput.value).toBe('Bar');
 	});
+
+	it('removes an option when click on remove button', () => {
+		const {container} = render(
+			<OptionsWithProvider
+				defaultLanguageId={themeDisplay.getLanguageId()}
+				editingLanguageId="pt_BR"
+				name="options"
+				onChange={jest.fn()}
+				spritemap={spritemap}
+				value={optionsValue}
+			/>
+		);
+
+		act(() => {
+			jest.runAllTimers();
+		});
+
+		const removeOptionButton = document.querySelector('.ddm-option-entry .close');
+
+		fireEvent.click(removeOptionButton);
+
+		act(() => {
+			jest.runAllTimers();
+		});
```

Comment:
@alinedoleron, do we need these two `jest.runAllTimers();`? Why the first one is necessary?

Reply:
Hi @natocesarrego, I followed the same way the others test were written. They always put this after the component creation and after the event.

Reply:
@alinedoleron, can you please find out why this is necessary?

Reply:
I won't block this pull just because of it, but I see this as an opportunity for you and I learn a bit more about what's going on behind the scenes. 😉 What do you think, Alininha?